tests/queries/0_stateless/03234_get_setting_or_default.sql

```diff
@@ -0,0 +1,26 @@
+SET custom_a = 'value_a';
+SET custom_b = 'value_b';
+SET custom_c = null;
+SET custom_d = 5;
+
+
+
+SELECT getSettingOrDefault('custom_a','default_a');
+SELECT getSettingOrDefault('custom_b','default_b');
+SELECT getSettingOrDefault('custom_c','default_c');
+SELECT getSettingOrDefault('custom_d','default_d');
```

Comment:
nit (here and further in the file)
```suggestion
SELECT getSettingOrDefault('custom_a', 'default_a');
SELECT getSettingOrDefault('custom_b', 'default_b');
SELECT getSettingOrDefault('custom_c', 'default_c');
SELECT getSettingOrDefault('custom_d', 'default_d');
```

Reply:
Done, thanks!